1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,5 +1,26 @@
-function setAlarm() {}
+function setAlarm() {
+  let timeDisplay = document.getElementById("timeRemaining");
+  let inputTime = document.getElementById("alarmSet");
+  let seconds = inputTime.value;
+  inputTime.value = "";
+  timeDisplay.innerText = `Time Remaining: ${timeFormatter(seconds)}`;
+  let countDownInterval = setInterval(() => {
+    seconds--;
+    timeDisplay.innerText = `Time Remaining: ${timeFormatter(seconds)}`;
+    if (timeDisplay.innerText === "Time Remaining: 00:00") {
```

Comment:
What will you need to remember to do here if you change the output of `timeFormatter` to eg `${minutes}m ${seconds}s`? Can you think of a different way to achieve exactly the same conditional check?